src/canRender.ts

```diff
@@ -1,12 +1,31 @@
+// Banners
+// --------------------------------------------
 import {
     COMPONENT_NAME as APP_BANNER_NAME,
     canRender as appBannerCanRender,
 } from './AppBanner/canRender';
+
 import {
     COMPONENT_NAME as DIGITAL_SUBSCRIBER_APP_BANNER_NAME,
     canRender as digitialSubCanRender,
 } from './DigitalSubscriberAppBanner/canRender';
+
+import {
+    COMPONENT_NAME as BANNER_WITH_LINK_NAME,
+    canRender as bannerWithLinkCanRender,
+} from './BannerWithLink/canRender';
+
+// Default Epics
+// --------------------------------------------
 import { COMPONENT_NAME as EPIC_NAME, canRender as epicCanRender } from './Epic/canRender';
+
+import {
+    COMPONENT_NAME as EPIC_WITH_IMAGE_NAME,
+    canRender as epicWithImageCanRender,
+} from './EpicWithSpecialHeader/canRender';
+
+// Old newsletter Epics
```

Comment:
Totally fine to organise things this way, but I'm wondering if we need to separate out the newsletter epics into old and new? I think it's just the name format which has changed between the two groups?

Reply:
In a few weeks we'll be moving to a more generic self-serve epic which should get rid of the need for Epic Newsletter naming strategies